File: chobobdev/0001.go
func solution(n int) string {
	var answer string
	if n%2 == 0 {
		for i := 0; i < n/2; i++ {
			answer += "수박"
		}
	} else {
		for i := 0; i < (n-1)/2; i++ {
			answer += "수박"
		}
		answer += "수"
	}

	return answer
}

File: chobobdev/0001.java
// 1 번 답안
class Solution {
  public String solution(int n) {
      String answer = "";
      String watermel = "수박";
      if(n%2==0){
          answer=watermel.repeat(n/2);
      }
      else{
          answer=watermel.repeat(n/2)+"수";
      }
      return answer;
  }
}